### PR TITLE
Tests for reading floating point values (including NaN, infinity)

### PR DESCRIPTION
**Pull request by dirkvdb**
As discussed in [https://github.com/rbock/sqlpp11/issues/184](url)

### CODE REVIEW

File: tests/FloatingPointTest.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2013 - 2016, Roland Bock
```
Comment (rbock) [inline tests/FloatingPointTest.cpp]: You might want to add your name and update the year.

Comment (dirkvdb) [reply tests/FloatingPointTest.cpp]: I hereby grant you full copyright. Consider it a gift :-)

Comment (rbock) [reply tests/FloatingPointTest.cpp]: Thanks :-)

**Comment by rbock** [comment]
Merged, thanks!

**Pull request merged**